src/app/personside/visittkort/body/Epost.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import Undertekst from 'nav-frontend-typografi/lib/undertekst';
+
+import VisittkortElement from './VisittkortElement';
+import { Kontaktinformasjon } from '../../../../models/person';
+
+const emailPath = require('../../../../resources/svg/email.svg');
```

Comment:
diskuterte dette med eigil her om dagen, og kom egentlig ikke på noe godt argument for å samle alle svg'er i en svg-mappe nå som vi bruker react og require. Jeg endte opp med å flytte svg'en til samme mappe som den brukes i, da blir hvertfall importen ryddigere, dessuten blir det klart hvor den brukes.

Reply:
og kanskje bytte navn til emailLogo istedenfor path?